plutus-core/cost-model/data/builtinCostModel.json

```diff
@@ -302,5 +302,19 @@
             },
             "type": "linear_cost"
         }
+    },
+    "blake2b": {
+        "memory": {
+            "arguments": 0,
+            "type": "constant_cost"
+        },
+        "cpu": {
+            "arguments": {
+                "slope": 0,
+                "intercept": 0,
+                "orientation": "x"
+            },
+            "type": "linear_cost"
+        }
```

Comment:
I think this can be removed.  The data in here is auto-generated from the benchmark results by `update-cost-model` and then read in later to create `defaultCostModel`; however, the `mempty` in `Builtins.hs` means that we'll never actually try to read this information, so it should be safe to delete it for now.

The way all this stuff works is quite complicated and I don't know if it's written down anywhere.  I'll add a description to the cost model document.

Reply:
I tried to remove it, but because I have some other boilerplate code , I get 
```
   • Error in $: parsing PlutusCore.Evaluation.Machine.BuiltinCostModel.BuiltinCostModelBase(BuiltinCostModelBase) failed, key "blake2b" not found
    • In the Template Haskell splice
        $$(readJSONFromFile DFP.builtinCostModelFile)
      In the expression: $$(readJSONFromFile DFP.builtinCostModelFile)
      In an equation for ‘defaultBuiltinCostModel’:
          defaultBuiltinCostModel
            = $$(readJSONFromFile DFP.builtinCostModelFile)
   |
36 |   $$(readJSONFromFile DFP.builtinCostModelFile)
 ```
 I put the stub json back.

Reply:
Ah, OK.  That's fine then.  It won't do any actual harm to have this here for the time being.